prime-router/docs/how-to-onboard-a-receiver.md

```diff
@@ -41,16 +52,24 @@ Below is an example of the organization file
         receivingFacilityOID:
 ```
 
-In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything that matches the code LT.
+In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything 
+that matches the code LT. Filters can be applied to the organization or receiver. For more information on filters see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how-to-use-filters.md)
 
-In addition, there is the translation section, which specifies the output format that will be sent to the receiver. Currently, we have three formats available:
+In addition, there is the translation section, which specifies the output format that will be sent to the receiver. 
+Currently, we have three formats available:
 
-- HL7 - Each record is converted to HL7 before being sent to the receiver
-- CSV - The records are converted to CSV before being sent to the receiver
+- HL7
+- CSV
+- FHIR
```

Comment:
Is FHIR available for the covid pipeline? I know CSV is not available for UP.

Reply:
good point. We did agree that these documents were specific to the covid pipeline. Removed FHIR